source/directives/youtube.rst

```diff
@@ -0,0 +1,97 @@
+YouTube and Vimeo
+=================
+The ``youtube`` directive embeds content from http://www.youtube.com directly in a page.
+
+The ``vimeo`` directive embeds content from http://www.vimeo.com directly in a page.
+
+Synopsis
+--------
+The general format of the ``youtube`` directive is:
+
+.. code-block:: rst
+
+   .. youtube:: youtube video id
+      :options:
+
+The general format of the ``vimeo`` directive is:
+
+.. code-block:: rst
+
+   .. vimeo:: vimeo video id
+      :options:
+
+Both the ``vimeo`` and ``youtube`` directives take the same options.
+
+Required Arguments
+------------------
+
+id
+    This must be the ID defined for this video by YouTube or Vimeo, respectively.
+
+content area
+    Neither the ``vimeo`` or ``youtube`` directive use a content area.
+
+Optional Arguments
```

Comment:
youtube now has a :divid: option , this is for backward compatibility when converting html5 hosted videos to the youtube platform.  Or just to use if you want to use a better descriptor to reference rather than the youtube id.

Reply:
I took a crack at documenting, but I don't actually get this option.  Is it possible to explicitly refer to a div in runestone somehow?  Is there a use case or example for this?

Is there a reason why Vimeo and `video` do not have this option?